apps/nextjs-website/react-components/types/VideoImage/VideoImage.types.ts

```diff
@@ -0,0 +1,50 @@
+import { CommonProps } from '../common/Common.types';
+
+export interface VideoImageProps extends CommonProps {
+  src: string | { url: string; mime: string };
+  alt: string;
```

Comment:
```suggestion
  src: { url: string; mime: string; alt?: string };
```
Remove `string` option and move `alt` into `src`

`alt` currently optional due to it being useless if media is a video